src/components/oscal-utils/OSCALLinkUtils.js

```diff
@@ -43,7 +43,6 @@ function getFileExtension(url) {
   try {
     return new URL(url).pathname.split("/").pop().split(".").pop();
   } catch {
-    console.warn(`Unable to detect extension for: ${url}`);
```

Comment:
I think I would rather not include this in the code base, especially with `console` statements causing linter warning. Is this necessary for the user to see?

Reply:
I agree. I do not see much benefit to the user from leaving console warnings in the codebase.